fabric-compute-framework/src/main/java/com/olacabs/fabric/compute/processor/ProcessorBase.java

```diff
@@ -35,9 +35,10 @@
 public abstract class ProcessorBase {
     @Getter
     private final boolean isScheduled;
+
     @Getter
     @Setter
-    private int id;
+    private String id;
```

Comment:
Ain't this used for acking? Also why is string necessary? You can just do Integer.toString() when setting into mdc